src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -100,6 +100,15 @@ class LocalBackend extends Backend with SharedFileSystem {
     }
   } map CompletedExecutionHandle
 
+  def useCachedCall(cachedBackendCall: BackendCall, backendCall: BackendCall)(implicit ec: ExecutionContext): Future[ExecutionHandle] = Future {
+    Try(cachedBackendCall.callRootPath.copyTo(backendCall.callRootPath)) match {
+      case Success(_) =>
```

Comment:
Sorry, didn't notice this was a multi-statement case.

But unrelated followup, why is the first thing done w/ the result of postProcess to call a .get on it?